app/api/resources/user.py

```diff
@@ -129,7 +129,14 @@ def get(cls):
     @users_ns.doc("update_user_profile")
     @users_ns.expect(auth_header_parser, update_user_request_body_model)
     @users_ns.response(HTTPStatus.OK, "%s" % messages.USER_SUCCESSFULLY_UPDATED)
-    @users_ns.response(HTTPStatus.BAD_REQUEST, "Invalid input.")
+    @users_ns.response(
+        HTTPStatus.BAD_REQUEST,
+    "%s\n%s"
+    % (
+         "Invalid input.",
+         messages.USERNAME_FIELD_IS_EMPTY
+    ),
+)
```

Comment:
Prettifying this makes this more unreadable :)

let's just go with the orignal style.